distribution/lib/Standard/Base/0.0.0-dev/src/Main.enso

```diff
@@ -47,7 +47,8 @@ from project.Data.Number.Extensions export all hiding Math, String, Double
 from project.Data.Noise export all hiding Noise
 from project.Data.Pair export Pair
 from project.Data.Range export Range
-from project.Data.Text.Extensions export Text, Split_Kind, Line_Ending_Style
+# TODO [RW] do we want to export case? I don't want to pollute too much, but without autoscoping to_case is unusable as you'd need to import it.
```

Comment:
Lets add a tracking ticket for autoscoping so this gets reviewed as part of that.